openedx/core/djangoapps/credit/views.py

```diff
@@ -0,0 +1,292 @@
+"""
```

Comment:
Note to reviewers: throughout this file, I have chosen not to internationalize the error messages returned by the API.  I expect these errors to be used for debugging purposes (especially for external credit providers integrating with us) and not shown to end-users.